test/truffle/docker/README.md

```diff
@@ -24,3 +24,12 @@ using.
 
 Docker will need to run the container with at least 8 GB of RAM if you are using
 virtualisation, to give enough space for the native image to build.
+
+## Ruby Managers
+
+There are also tested by Dockerfiles. These all run on Ubuntu. We test rbenv,
```

Comment:
There => They but I think it'd be clearer to use 
Ruby Managers are also tested by Dockerfiles.

Reply:
Fixed.